doc/source/docs/user_guide/data_structures.rst

```diff
@@ -48,20 +48,20 @@ element of the series with that geometry.  In either case, a
 :class:`~pandas.Series` or a :class:`GeoSeries` will be returned, as appropriate.
 
 A short summary of a few attributes and methods for GeoSeries is
-presented here, and a full list can be found in the :doc:`all attributes and methods page <../reference/geoseries>`.
+presented here, and a full list can be found in the :doc:`GeoSeries <../reference/geoseries>` API reference.
```

Comment:
```suggestion
presented here, and a full list can be found in the :doc:`GeoSeries API reference <../reference/geoseries>`.
```

While this might not exactly follow the rules then, I think it is a bit better to include "API refererce", otherwise it can look like a link to the GeoSeries docstring page.